File: tmdb-core/src/main/java/dev/reuss/tmdb/domain/find/FindPaths.java
package dev.reuss.tmdb.domain.find;


import dev.reuss.tmdb.value.id.ExternalId;

final class FindPaths {

    private static final String FIND = "/find";

    private FindPaths() {
    }

    static String byExternalId(ExternalId externalId) {
        return FIND + "/" + externalId.value();
    }
}

File: tmdb-core/src/main/kotlin/dev/reuss/tmdb/domain/find/FindPaths.kt
package dev.reuss.tmdb.domain.find

import dev.reuss.tmdb.core.path.tmdbPath
import dev.reuss.tmdb.value.id.ExternalId

internal object FindPaths {
    fun byExternalId(externalId: ExternalId): String = tmdbPath("find", externalId)
}
